### PR TITLE
Updating snippets and samples to use latest version of Rabbit beta5

### PR DESCRIPTION
**Pull request by WojcikMike**
cc: @Particular/docs-maintainers when green please merge

### CODE REVIEW

File: Snippets/Rabbit/Rabbit_All/NativeSendTests.cs [hunk 1]
```diff
@@ -55,7 +54,7 @@ Task<IEndpointInstance> StartBus()
         endpointConfiguration.SetTypesToScan(typesToScan);
         endpointConfiguration.EnableInstallers();
         endpointConfiguration.UsePersistence<InMemoryPersistence>();
-        endpointConfiguration.DisableFeature<SecondLevelRetries>();
+        endpointConfiguration.Recoverability().Delayed(d => d.NumberOfRetries(0));
```
Comment (SimonCropp) [inline Snippets/Rabbit/Rabbit_All/NativeSendTests.cs]: this is prob redundant for the test?

**Comment by SimonCropp** [comment]
one comment. but the rest looks good

**Comment by WojcikMike** [comment]
Comment addressed

**Comment by WojcikMike** [comment]
@SimonCropp build is green, sill you do the honors?

**Pull request merged**